### PR TITLE
Addition of Australia OCDID's for District, Region Electorates for State Legislative Assembly and Councils

### PR DESCRIPTION
**Pull request by GoogleCivicsKantar**
Hi Novita,

This Pull Request contains below added/updated files for Australia

1. Added assemblies.csv which represents the district electorates and region electorates for state legislative assembly.
2. Added councils.csv which represents the district electorates and region electorates for state legislative council.
3. Updated country-au.csv with new ocdid's.
4. Updated README file with newly added csv files summary.

### CODE REVIEW

**Comment by GoogleCivicsKantar** [comment]
Hi @jloutsenhizer , Can you please review these OCDID's for Australia? I have added assemblies.csv and councils.csv.

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
Thanks for putting this together. I've checked the number of OCD IDs for assemblies and councils and they look correct, but I haven't checked the individual constituencies.

I had one comment about the types used

File: identifiers/country-au/assemblies.csv [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+ocd-division/country:au/state:wa/district:balcatta,Balcatta
+ocd-division/country:au/state:wa/district:baldivis,Baldivis
+ocd-division/country:au/state:nsw/district:ballina,Ballina
+ocd-division/country:au/state:nsw/district:balmain,Balmain
+ocd-division/country:au/state:qld/district:bancroft,Bancroft
+ocd-division/country:au/state:nsw/district:bankstown,Bankstown
+ocd-division/country:au/territory:nt/district:barkly,Barkly
+ocd-division/country:au/state:qld/district:barron_river,Barron River
+ocd-division/country:au/state:nsw/district:barwon,Barwon
+ocd-division/country:au/state:tas/region:bass,Bass
```
[the remark this answers is not in the document]
Comment (Nishagowda29) [reply identifiers/country-au/assemblies.csv:24]: Hii @jloutsenhizer,

The Australian Statistical Geography Standard (ASGS) provides a framework of statistical areas used by the Australian Bureau of Statistics (ABS) and other organization’s to enable the publication of statistics that are comparable and spatially integrated. So these are the set of standard areas provided by ASGS that can be used for analyzing, visualizing and integrating statistics and various other useful studies.

The ASGS defines two types of spatial structures:

ABS Structures -- statistical areas designed specifically for outputting statistics
Non-ABS Structures -- administrative areas, e.g. local government areas (LGAs) and postal areas (POAs)
 
State Electoral districts(SEDs) comes under Non-ABS structure of ASGS, further under which districts and regions are defined under SED’s. Further, as shown in the attached table, the majority of the cases under the column Council(upper house) has multimember constituencies which are defined as regions and single member constituencies as districts. However, this nomenclature is not the followed everywhere as there is a immediate change of usage in the Adjoining Assembly(Lower house) column.

For example: Tasmania have 5 regions which comes under Assembly(Lower house) and having multimember constituency, but at the same time even Australian Capital Territory (ACT) have 5 electoral districts which also have multimember constituency.

[Australia_OCD_ID.xlsx](https://github.com/opencivicdata/ocd-division-ids/files/10215012/Australia_OCD_ID.xlsx)

So, with this example differentiating and dividing all of them under SLDL and SLDU will be highly difficult and will also not be able to provide the justice to any of them, as electoral districts/region are uniquely defined for different states.

**Note: Pls find the link from where all the above details have been considered for our understanding.** 
(https://rpubs.com/Earlien/ASGS_and_Non-ASGS_Spatial_Structures)

[the remark this answers is not in the document]
Comment (jloutsenhizer) [reply identifiers/country-au/assemblies.csv:24]: Sorry for the delay in responding to this.

I'm not sure if I'm fully following the differences between regions and districts. Specifically I'm not sure I understand what specific information is conveyed by knowing that a specific electoral constituency is a region or a district. As most regions are mutli-member constituencies, but in some cases districts are also multi-member constituencies as we pointed out for the Australian Capital Territory.

What I'm suggesting is that we use what level of government the electoral constituency elects members to as the decider for the type of the consituency.

So:
- All of the regions/districts that elect member(s) to a lower (or unicameral) house would be `sldl` 
- All of the regions/districts that elect member(s) to an upper house would be `sldu`

Let me know if I'm missing something here.

[the remark this answers is not in the document]
Comment (Nishagowda29) [reply identifiers/country-au/assemblies.csv]: Hi @jloutsenhizer,
As suggested, for uniformity we have changed lower house district/regions to sldl and upper house district/region as sldu. 
Please let us know if any further changes are required and also let us know if incase you have any concern on the same.

File: identifiers/country-au/README [hunk 2]
```diff
@@ -3,8 +3,12 @@ README
...
+*   assemblies.csv: represents the district electorates and region electorates for state Legislative Assembly
+*   councils.csv: represents the district electorates and region electorates for state Legislative Council
 
 OCD ID Types used used:
 *   federal_electorate : represents the electorates for the Australian House of Representatives
 *   state: states in Australia
-*   territory: territories in Australia
\ No newline at end of file
+*   territory: territories in Australia
+*   district: districts in Australia
```
Comment (jloutsenhizer) [inline identifiers/country-au/README:13]: Please update the README to reflect the new types

**Review by jloutsenhizer — APPROVED** [review]
Just one more nitpick to the README, but otherwise this looks good to me.

File: identifiers/country-au/README (hunk 2 above)
Comment (GoogleCivicsKantar) [reply identifiers/country-au/README]: Hi @jloutsenhizer, As suggested, we have updated README file.

File: identifiers/country-au/README [hunk 3]
```diff
@@ -3,8 +3,12 @@ README
...
+*   councils.csv: contains the state legislative council electorates (sldu)
 
 OCD ID Types used used:
 *   federal_electorate : represents the electorates for the Australian House of Representatives
 *   state: states in Australia
-*   territory: territories in Australia
\ No newline at end of file
+*   territory: territories in Australia
+*   sldl: state legislature distict lower
+*   sldu: state legislature distict upper
```
Comment (jpmckinney) [inline identifiers/country-au/README:14]:
```suggestion
*   sldl: state legislature district lower
*   sldu: state legislature district upper
```

**Review by jpmckinney — COMMENTED** [review]
Looks good from a quick glance, minus a typo.

File: identifiers/country-au/assemblies.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (jpmckinney) [reply identifiers/country-au/assemblies.csv]: Hmm, now that I look at https://en.wikipedia.org/wiki/Electoral_districts_of_Western_Australia I'm not sure that districts and regions should be mixed. Districts are *within* regions. The [Western Australian Legislative Assembly](https://en.wikipedia.org/wiki/Western_Australian_Legislative_Assembly) elects "from single-member [electoral districts](https://en.wikipedia.org/wiki/Electoral_districts_of_Western_Australia)" and so regions don't even come into play. If no state cares about regions for electoral outcomes except as administrative trivia, we have two options:

* Keep `region:` and nest `sld*:` under it (NB: Tasmania calls them "divisions")
* Use only `sld*:` and skip all regions

[the remark this answers is not in the document]
Comment (jpmckinney) [reply identifiers/country-au/assemblies.csv]: Thanks, @Nishagowda29. Sorry, I misread the files (I thought I saw regions and districts being mixed). Looks good on second reading.

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**